packages/react-core/src/components/Form/FormGroup.tsx

```diff
@@ -40,24 +45,34 @@ export const FormGroup: React.FunctionComponent<FormGroupProps> = ({
   hasNoPaddingTop = false,
   helperText,
   helperTextInvalid,
+  helperTextIcon,
+  helperTextInvalidIcon,
   fieldId,
   ...props
 }: FormGroupProps) => {
-  const validHelperText = (
-    <div
-      className={css(styles.formHelperText, validated === ValidatedOptions.success && styles.modifiers.success)}
-      id={`${fieldId}-helper`}
-      aria-live="polite"
-    >
-      {helperText}
-    </div>
-  );
+  const validHelperText =
+    helperText && (helperText as React.ReactElement).type === FormHelperText ? (
```

Comment:
why is the type check needed? i think this would break if for whatever reason the user wrapped like this
```
helperText={
<>
  < FormHelperText />
</>
}
```

Reply:
It would double wrap it if that was passed in I think. Would it be better to check if it is passed a string to wrap it, and if it gets a Node, it just displays it?

Reply:
I think that better matches what we do in other components and is a good alternative to what I suggested in my review

Reply:
Updated the check